build.sbt

```diff
@@ -79,8 +81,10 @@ lazy val `core-common` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
       "org.typelevel" %%% "cats-mtl" % CatsMtlVersion,
       "org.typelevel" %%% "vault" % VaultVersion,
       "org.typelevel" %%% "cats-laws" % CatsVersion % Test,
-      "org.typelevel" %%% "discipline-munit" % "2.0.0-M2" % Test,
+      "org.typelevel" %%% "discipline-munit" % DiscplineMUnitVersion % Test,
       "org.scalameta" %%% "munit" % MUnitVersion % Test
```

Comment:
oof. we should really change the scalafmt config to allow trailing commas

Reply:
The one I copied around for years used to, and then it broke. :shrug:

Reply:
D: lemme see what the current config value for it is

Reply:
https://scalameta.org/scalafmt/docs/configuration.html#trailing-commas

it's a little confusing. it seems like we'd want `rewrite.trailingCommas.style = "keep"`. I'm not clear on why `"multiple"` exists as an option though, so I might be misunderstanding something